packages/ceramic-cli/src/@types/identity-wallet.d.ts

```diff
@@ -1,8 +1,6 @@
 
 declare module 'identity-wallet' {
   export default class IdentityWallet {
-    constructor (getConsent: () => Promise<boolean>, opts: any)
```

Comment:
Not related to this PR but we should export the types in IDW so it's not needed to create custom declarations for it.

Reply:
👍